index.js

```diff
@@ -1,57 +1,84 @@
-const form = document.getElementById('myForm');
-const container = document.getElementById('container');
-const bookTitle = document.getElementById('title');
-const bookAuthor = document.getElementById('author');
-const books = JSON.parse(localStorage.getItem('books')) || [];
-
-const addBookToLocalStorage = (id, title, author) => {
-  if (title && author) {
-    books.push({
-      id,
-      title,
-      author,
-    });
+/* eslint-disable max-classes-per-file */
+
+class Book {
+  constructor(title, author) {
+    this.title = title.value;
+    this.author = author.value;
+  }
+}
+
+class LocalStoragConfig {
+  static getBookFromStorage() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static AddBook(book) {
+    const books = LocalStoragConfig.getBookFromStorage();
+    books.push(book);
     localStorage.setItem('books', JSON.stringify(books));
-    return { id, title, author };
   }
-  return null;
-};
-
-const createBookUi = ({ title, author, id }) => {
-  const newDiv = document.createElement('div');
-  newDiv.classList.add('content');
-  container.appendChild(newDiv);
-  newDiv.innerHTML += `
-        <h2 class="title">${title}</h2>
-        <h3 class="author">${author}</h3>
-        <button class="remove" name=${id}>Remove</button>
-        <div class="line"></div>
-    `;
-};
-
-books.forEach(createBookUi);
-form.onsubmit = (e) => {
+
+  static RemoveStoring(title) {
```

Comment:
- [ ] Good job so far on removing the book but I have noticed that when we remove a book it is only removed on the UI but still stored in the Localstorage so when I reload the page even the books I removed display again on the UI, Kindly fix that

https://user-images.githubusercontent.com/75126481/149295310-082e6562-1343-4330-86b8-068a52e5de23.MP4